doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -337,35 +226,35 @@ The examples below demonstrate the use of these variables and methods.
 Set component definition and memory ranges
 ------------------------------------------
 
-In the :file:`app_envelope_yaml.jinja2` (found `here <https://github.com/nrfconnect/suit-generator/blob/main/ncs/app_envelope.yaml.jinja2>`__), the component definition and memory ranges are filled out by using the ``edtlib`` object like so:
+In the :file:`app_envelope_yaml.jinja2` (found `here <https://github.com/nrfconnect/suit-generator/blob/v2.7.0/ncs/app_envelope.yaml.jinja2>`__), the component definition and memory ranges are filled out by using the ``edtlib`` (devicetree values) object like so:
```

Comment:
Is it okay that we are using a link here? The page doesn't exist yet.
Maybe it should point to this instead?
```
`:file:`modules/lib/suit-generator/ncs/app_envelope.yaml.jinja2`
```

Reply:
Changed to "file"